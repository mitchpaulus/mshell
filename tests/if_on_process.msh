# This is running the process command false. The '?' pushes the exit code on the stack when finished.
# 0 results in the true branch in mshell.
["false"]?
("The command run had a successful exit code" wl)
("The command run had an unsuccessful code" wl)
iff
