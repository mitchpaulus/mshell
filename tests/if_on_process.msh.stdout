The command run had an unsuccessful code
